tests/serverpod_test_server/generated/tables.pgsql

```diff
@@ -84,7 +97,8 @@ CREATE TABLE "types" (
   "aDouble" double precision,
   "aDateTime" timestamp without time zone,
   "aString" text,
-  "aByteData" bytea
+  "aByteData" bytea,
+  "anUri" character varying
```

Comment:
This should be of type `text` to be consistent with other database fields. (`text` and `varchar` are interchangeable in Postgres)

Reply:
Okk i will change it. Please LMK anything else.